subjects/printascii/README.md

```diff
@@ -0,0 +1,27 @@
+## print-ascii
+
+### instructions
+
+Write a program that prints the ASCII value of a character passed as an argument to the program
```

Comment:
I would consider changing this part to "Write a program that prints the ASCII value of a letter passed". Also, it is said bellow that the program should print (`'\n'`) at the end of the output. Does that apply to this case?

Reply:
fixed